totest-manager.py

```diff
@@ -206,7 +208,7 @@ def all_repos_done(self, project, codes=None):
         for repo in root.findall('result'):
             # ignore ports. 'factory' is used by arm for repos that are not
             # meant to use the totest manager.
-            if repo.get('repository') in ('ports', 'factory'):
+            if repo.get('repository') in ('ports', 'factory', 'images_staging'):
```

Comment:
But actually we are excluding this for all `arch`. Only ARM is using `images_staging` repo?